client/src/components/Kanban/Column/Column.tsx

```diff
@@ -1,14 +1,20 @@
 import { Box, Grid, Typography } from '@material-ui/core';
 import { Droppable } from 'react-beautiful-dnd';
 import { ICard } from '../../../interface/Card';
-import { IColumn } from '../../../interface/Column';
 import Card from '../Card/Card';
 import CardForm from '../CardForm/CardForm';
 import useStyles from './useStyles';
 
-type ColumnProps = IColumn;
+interface ColumnProps {
+  id: string;
+  name: string;
+  cards: ICard[];
+  createdAt?: Date;
+}
```

Comment:
Doing this `id` on `IColumn` is actually `_id`; keeping consistent with our frontend vs backend (can also consider changing prop to be `_id` too if need be)